src/openrct2/ride/Ride.cpp

```diff
@@ -5321,34 +5320,34 @@ Vehicle* RideGetBrokenVehicle(const Ride& ride)
  *
  *  rct2: 0x006D235B
  */
-void Ride::Delete()
+void Ride::remove()
 {
     RideDelete(id);
```

Comment:
I wonder if we should rename this function to be consistent with `Ride::Delete` being renamed to `Ride::remove` as well.

Reply:
Your call. I’m fine either way.

Reply:
To my surprise, this seems to be the only place `RideDelete` is ever called. We could just integrate it into the class, I suppose. I don't want to move the goal posts for this PR, though.

Reply:
In that case, could you approve it?